docs/source/index.rst

```diff
@@ -77,6 +77,7 @@ distributed
 .. automodule:: torch_xla.distributed.parallel_loader
 .. autoclass:: ParallelLoader
```

Comment:
Let's remove `ParallelLoader` from the docs. You should almost never use it instead of `MpDeviceLoader`